Blockzilla/BrowserViewController.swift

```diff
@@ -1610,6 +1610,7 @@ extension BrowserViewController: OverlayViewDelegate {
         if url == nil {
             Telemetry.default.recordEvent(category: TelemetryEventCategory.action, method: TelemetryEventMethod.typeQuery, object: TelemetryEventObject.searchBar)
             Telemetry.default.recordSearch(location: .actionBar, searchEngine: searchEngineManager.activeEngine.getNameOrCustom())
+            recordSearchEvent(.suggestion)
```

Comment:
Oh, was this actually missing as well? Good catch.

Reply:
Yes, not ideal. The search has many entry points.